osbs/build/config_map_response.py

```diff
@@ -34,10 +35,23 @@ def get_data(self):
         """
         Find the data stored in the config_map
 
-        :return: dict, the json data that was passed into the ConfigMap on creation
+        :return: dict, the data that was passed into the ConfigMap on creation
         """
         data = graceful_chain_get(self.json, "data")
         if data is None:
             return {}
 
         return data
+
+    def get_json(self):
+        """
+        Find the data stored in the config_map
+
+        :return: json, the json version of the json data that was passed into the ConfigMap on
+                 creation
+        """
+        data = graceful_chain_get(self.json, "data")
+        if data is None:
+            return {}
+
+        return json.dumps(data)
```

Comment:
Well, we should improve the test data so that it tries to store structured data there. In mock_jsons/*/create_config_map.json we'd want:

```json
...
"data": {
    "metadata.json": "{\"key\": \"value\"}",
    "special.how": "\"very\"",
    "special.type": "\"charm\""
}
...
```

and `test_config_map()` should be given the additional "metadata.json" data to attempt to store.

Finally, `test_config_map()` should verify the behaviour of `config_map.get_json()`, i.e. that you get back the same objects you put in, for each key ("metadata.json", "special.how", whatever).